pkgs/by-name/et/etterna/package.nix

```diff
@@ -0,0 +1,109 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
+  copyDesktopItems,
+  # deps
+  cmake,
+  pkg-config,
+  openssl,
+  libGLU,
+  xorg,
+  alsa-lib,
+  libjack2,
+  libpulseaudio,
+  libogg,
+  sse2neon,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "etterna";
+  version = "0.74.3";
+
+  src = fetchFromGitHub {
+    owner = "etternagame";
+    repo = "etterna";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-zzCk6axISswfTAk7rRha5HFzIHQ0AjpAZyAWzH+Cn1s=";
+  };
+
+  patches = [ ./fix-download-manager.patch ];
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+
+    copyDesktopItems
+  ];
+
+  buildInputs = [
+    openssl
+    alsa-lib
+    libjack2
+    libpulseaudio
+    libGLU
+    libogg
+    sse2neon
+    xorg.libXinerama
+    xorg.libXrandr
+    xorg.libX11
+    xorg.libXext # Needed for DPMS
+    xorg.libXvMC
+  ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "etterna";
+      desktopName = "Etterna";
+      genericName = "Rhythm and dance game";
+      icon = "etterna";
+      tryExec = "etterna";
+      exec = "etterna";
+      categories = [
+        "Application"
+        "Game"
+        "ArcadeGame"
+      ];
+      comment = "A cross-platform rhythm video game.";
+      terminal = false;
+    })
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/{bin,share/etterna}
+    mkdir -p $out/share/applications
+    # copy select necessary game files into virtual fs
+    cp -r /build/source/{Announcers,Assets,BGAnimations,BackgroundEffects,BackgroundTransitions,Data,GameTools,NoteSkins,Scripts,Themes} "$out/share/etterna/$dir"
+
+    # copy binary
+    cp /build/source/Etterna $out/bin/etterna-unwrapped
```

Comment:
I assume this makes `etterna-unwrapped` available in the user's `$PATH`, which might not be wanted

Reply:
Is there a way to not make it available while also keeping the wrapped version working?

Reply:
The usual "fix" is to name the unwrapped binary `.etterna-unwrapped` (this is what the regular wrapProgram would do)